txnkv/transaction/2pc.go

```diff
@@ -775,6 +777,9 @@ func (c *twoPhaseCommitter) doActionOnGroupMutations(bo *retry.Backoffer, action
 	// Already spawned a goroutine for async commit transaction.
 	if actionIsCommit && !actionCommit.retry && !c.isAsyncCommit() {
 		secondaryBo := retry.NewBackofferWithVars(c.store.Ctx(), CommitSecondaryMaxBackoff, c.txn.vars)
+		if c.store.IsClose() {
+			return tikverr.ErrStoreIsExiting
```

Comment:
I don't think we should return an error. Failing to commit secondary locks does not mean the transaction fails.
If we return an error here, the user may receive a commit failure when the transaction succeeds to commit.

Reply:
Done